test/converters/converters.go

```diff
@@ -0,0 +1,21 @@
+package converters
+
+import (
+	"github.com/bitrise-io/steps-deploy-to-bitrise-io/test/converters/junitxml"
+	"github.com/bitrise-io/steps-deploy-to-bitrise-io/test/converters/xcresult"
+)
+
+type Intf interface {
+	SetFiles([]string)
```

Comment:
i would remove this func, why does it worth to hold the files if detect may filters out a couple of them?
pass the files to detect